src/commands/version.cr

```diff
@@ -0,0 +1,28 @@
+require "../helpers/versions"
+require "./command"
+
+module Shards
+  module Commands
+    class Version < Command
+      def self.run(path, *args)
+        # look up for `shard.yml` in given directory or up
```

Comment:
Was thinking on that, but that would be a another class method, so perhaps is better move that to a helper instead?